src/Components/CheatSheet.js

```diff
@@ -14,20 +21,67 @@ class CheatSheet extends Component {
         return resp.json();
       })
       .then(data => {
-        // console.log(data.kana);
-        this.handleShowData(data.kana);
+        this.setState({
+          kanaTable: data.kana
+        });
       });
   };
 
-  handleShowData = data => {
-    console.log(data);
+  handleGetHiragana = () => {
+    this.setState({
+      isHiraganaShown: true,
+      isKatakanaShown: false
+    });
+  };
+  handleGetKatakana = () => {
+    this.setState({
+      isHiraganaShown: false,
+      isKatakanaShown: true
+    });
   };
 
   render() {
+    const { isHiraganaShown, isKatakanaShown, kanaTable } = this.state;
     return (
       <section className="cheat-sheet" onClick={this.handleFetchData}>
         <UserNavBar />
-        <main className="cheat-sheet-container"></main>
+        <main className="cheat-sheet-container">
+          <div className="cheat-sheet-btns">
+            <button
+              className="cheat-sheet-hiragana-btn"
+              onClick={this.handleGetHiragana}
+            >
+              Hiragana
+            </button>
+            <button
+              className="cheat-sheet-katakana-btn"
+              onClick={this.handleGetKatakana}
+            >
+              Katakana
+            </button>
+          </div>
+          {isHiraganaShown &&
+            kanaTable.map(kana => (
+              <SingleSign
+                kanaTable={kana.hiragana}
+                kanaMeaning={kana.meaning}
+                syllabary="hiragana"
+                key={kana.id}
+              />
+            ))}
+          {isKatakanaShown &&
+            kanaTable.map(kana => (
+              <SingleSign
+                kanaTable={kana.katakana}
+                kanaMeaning={kana.meaning}
+                syllabary="katakana"
```

Comment:
We no longer need to pass the syllabary prop here :) The same for line https://github.com/FAIRit/fe-kana-app/pull/6/files#diff-fa222d7b308a9775454bafcfe66605e1R68